Packages/com.unity.inputsystem/Documentation~/UserManagement.md

```diff
@@ -2,43 +2,45 @@
 
 ### Loss of Device
 
-If Input Devices which are paired to a user disconnect during the session, the system notifies the [`InputUser`](../api/UnityEngine.InputSystem.Users.InputUser.html) class. It still keeps track of the Device, and automatically re-pairs the Device if it becomes available again.
+If paired Input Devices disconnect during the session, the system notifies the [`InputUser`](../api/UnityEngine.InputSystem.Users.InputUser.html) class. It still keeps track of the Device, and automatically re-pairs the Device if it becomes available again.
 
-You can get notified of any such changes by subscribing to the [`InputUser.onChange`](../api/UnityEngine.InputSystem.Users.InputUser.html#UnityEngine_InputSystem_Users_InputUser_onChange) event.
+To get notifications about these changes, subscribe to the [`InputUser.onChange`](../api/UnityEngine.InputSystem.Users.InputUser.html#UnityEngine_InputSystem_Users_InputUser_onChange) event.
 
 ## User account management
 
-A user can be associated with a platform user account, if both the platform and the Devices used support this. Consoles commonly support this functionality. You can query the associated user account for an [`InputUser`](../api/UnityEngine.InputSystem.Users.InputUser.html) using the [`platformUserAccountHandle`](../api/UnityEngine.InputSystem.Users.InputUser.html#UnityEngine_InputSystem_Users_InputUser_platformUserAccountHandle) property. This property gets determined when the user is first [paired to a Device](#device-pairing), and the Device has any platform user information the Input System can query.
+The Input System can associate a user with a platform-specific user account, if both the platform and the Devices support this. Consoles commonly support this functionality. Platforms that support user account association are Xbox One, PlayStation 4, Nintendo Switch, and UWP.
+
+Use the [`platformUserAccountHandle`](../api/UnityEngine.InputSystem.Users.InputUser.html#UnityEngine_InputSystem_Users_InputUser_platformUserAccountHandle) property to query the associated user account for an [`InputUser`](../api/UnityEngine.InputSystem.Users.InputUser.html). This property gets determined when the user is first [paired to a Device](#device-pairing), and the Device has any platform user information the Input System can query.
 
 The account associated with an InputUser might change if the player uses the platform's facilities to switch to a different account ([`InputUser.onChange`](../api/UnityEngine.InputSystem.Users.InputUser.html#UnityEngine_InputSystem_Users_InputUser_onChange) receives an `InputUserChange.AccountChanged` notification).
 
-Platforms that support user account association are Xbox One, PlayStation 4, Nintendo Switch, and UWP. Note that for WSA/UWP apps, the *User Account Information* capability must be enabled for the app in order for user information to come through on input devices.
```

Comment:
Are we removing these because they're NDA platforms?

Reply:
No, I just moved these to the end of line 37.